### PR TITLE
Added display of email when scorer has no first/last name; hid other …

### PR DESCRIPTION
**Pull request by williln**
…scores when user's score isn't >0.

Addresses part of https://github.com/DjangoGirls/djangogirls/issues/86. User can still add score of None and comment of None, but you can no longer save the application with a None score and still see the other scores.

### CODE REVIEW

File: templates/application_detail.html [hunk 1]
```diff
@@ -39,7 +39,12 @@ <h3 class="page-header">Application #{{ application.number }} for {{ page.title
                 </thead>
                 {% for s in scores %}
                 <tr>
-                    <td>{{ s.user.first_name }} {{ s.user.last_name }}</td>
+                    <td>{% if s.user.first_name or s.user.last_name %}
```
Comment (aniav) [inline templates/application_detail.html]: I think this should be taken from `_str_` method on the user model :)

**Comment by williln** [comment]
@aniav Changed to use just s.user :smile: Now it looks like this: 

screen shot 2015-07-22 at 11 15 00 am

**Comment by aniav** [comment]
Well, it seems ok, I think. It could be improved on the model side but I think I will just merge it now :D

**Comment by williln** [comment]
I'm happy to keep working on it; I want to address @olasitarska's note about users shouldn't be able to save when both score and comments are null, anyway. What do you have in mind? __str__ should show name if there is a name, and email if there isn't?

**Comment by aniav** [comment]
Yeah, I think it should be checked and if both first name and last name are empty it should just siply display the e-mail, and I feel this can be changed in `_str_` as this is the behaviour we would always like to have for representing the user :)

**Pull request merged**